x-pack/test/functional/apps/saved_objects_management/import_saved_objects_between_versions.ts

```diff
@@ -27,19 +27,26 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       if (parts.length !== 3) {
         throw new Error('text not loaded yet');
       }
-      return Number.parseInt(parts[1], 10);
+      const count = Number.parseInt(parts[1], 10);
+      if (count === 0) {
+        throw new Error('text not loaded yet');
+      }
```

Comment:
The new failure occurs because before the ajax calls are performed, the label displays `exports 0 objects`. We're now waiting until the end of the call by checking that the numeric value is not zero (there will always be at least one object in the table, as the config object is present)